packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -318,16 +318,18 @@ test('lib-wallet issuer and purse methods', async (/** @type {LibWalletTestConte
         value: [],
         currentAmountSlots: {
           body: '{"brand":{"@qclass":"slot","iface":"Alleged: Zoe Invitation brand","index":0},"value":[]}',
-          slots: [{ kind: 'brand', petname: 'zoe invite' }],
+          slots: [
+            { boardId: 'board0223', kind: 'brand', petname: 'zoe invite' },
+          ],
         },
         currentAmount: {
-          brand: { kind: 'brand', petname: 'zoe invite' },
+          brand: { boardId: 'board0223', kind: 'brand', petname: 'zoe invite' },
           value: [],
         },
       },
       {
         brand: purseLog[1].brand,
-        brandBoardId: 'board0566',
+        brandBoardId: 'board0425',
```

Comment:
Why did this boardId change?

Reply:
Maybe because the order of the `getId` invocations changed? It seems correct though, as long as the board id is the same as that in the amount, and it's not the same as another brand.